demisto_sdk/commands/upload/uploader.py

```diff
@@ -268,7 +269,13 @@ def _upload_single(self, path: Path) -> bool:
             )
             self.failed_parsing.append((path, reason))
             return False
-
+        if (
+            self.marketplace
+            and self.marketplace not in content_item.marketplaces
+            and isinstance(content_item, ContentItem)
```

Comment:
```suggestion
            and isinstance(content_item, ContentItem)
            and self.marketplace not in content_item.marketplaces
```
should be switched since if its not the content_item is not `ContentItem` type, it would raise `AttributeError`